react/components/EditorContainer/Sidebar/index.tsx

```diff
@@ -33,14 +32,17 @@ const Sidebar: React.FunctionComponent<Props> = ({
   highlightHandler,
   intl,
   runtime,
+  visible,
 }) => {
   const formMeta = useFormMetaContext()
   const modal = useModalContext()
 
   return (
     <div
       id="sidebar-vtex-editor"
-      className="z-1 h-100 top-3em-ns calc--height-ns w-18em-ns w-100 w-auto-ns flex flex-row-reverse overflow-x-auto"
+      className={`z-1 h-100 top-3em-ns calc--height-ns w-18em-ns w-100 w-auto-ns ${
+        visible ? 'flex' : 'dn'
+      } flex-row-reverse overflow-x-auto`}
```

Comment:
I think that using `dn` makes all of the remaining classes unnecessary, so that could be:
```ts
className={
  visible
    ? 'z-1 h-100 top-3em-ns calc--height-ns w-18em-ns w-100 w-auto-ns flex flex-row-reverse overflow-x-auto'
    : 'dn'
 }
```